spec/lib/govuk_logging_spec.rb

```diff
@@ -4,32 +4,47 @@
 require "rack/test"
 
 RSpec.describe GovukLogging do
+  let(:govuk_headers_class) do
+    Class.new do
+      def self.headers
+        { govuk_request_id: "some-value" }
+      end
+    end
+  end
   before do
     stub_const("DummyLoggingRailsApp", Class.new(Rails::Application) do
       config.hosts.clear
       routes.draw do
         get "/error", to: proc { |_env| raise StandardError, "default exception" }
       end
     end)
+    stub_const("GovukHeaders", govuk_headers_class)
```

Comment:
```suggestion
```

I don't think you need this as you just use it for stubbing later which can be replaced.